config/creds/README.creds

```diff
@@ -0,0 +1,21 @@
+This directory will hold credentials files on the build host.
+
+All JSON files in this directory are ignored via gitignore(5) to reduce the
+risk that credentials accentally get commit.
+
+The following files are currently used:
+
+* gce.json
+
+  A service account JSON file for the Google Cloud Platform. This file can be
+  generated as describe here:
+  https://cloud.google.com/storage/docs/authentication#generating-a-private-key
```

Comment:
The Google API client is needed for GCE, which is a transitive dependency of the new ImageFactory and documented here:

https://github.com/redhat-imaging/imagefactory/blob/master/imagefactory_plugins/GCE/README.gce

I think our README can be cleaned up. Some of the dependencies that are mentioned are not needed anymore in recent imagefactory. I got some PRs accepted that turn some hard dependencies soft dependencies (which we don't need/use).

What if we agree that I'll simplify the README in a follow up PR (shortly, ~few weeks), would that be OK?